From 666da343fd8aea878728b04207792b1241904a20 Mon Sep 17 00:00:00 2001
From: "Choong, Yin Thong" <yin.thong.choong@intel.com>
Date: Wed, 24 Jul 2019 05:37:05 +0000
Subject: [PATCH] enable-test-platform-pthread

Signed-off-by: Choong, Yin Thong <yin.thong.choong@intel.com>
---
 test/test-platform/CMakeLists.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/test/test-platform/CMakeLists.txt b/test/test-platform/CMakeLists.txt
index 5c54b4a7..4749b27a 100644
--- a/test/test-platform/CMakeLists.txt
+++ b/test/test-platform/CMakeLists.txt
@@ -31,8 +31,11 @@ if(NETWORKING)
         main.cpp
         TestPlatform.cpp)
 
+	set(CMAKE_THREAD_PREFER_PTHREAD 1)
+	find_package(Threads REQUIRED)
+
     target_link_libraries(test-platform
-            PRIVATE parameter pfw_utility remote-processor)
+            PRIVATE parameter pfw_utility remote-processor Threads::Threads)
     # Workaround because asio is still leaking to test-platform
     target_link_libraries(test-platform PRIVATE asio)
 
-- 
2.21.0

